packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -93,6 +93,11 @@ export class BlobManager {
         const storageId = this.redirectTable?.get(blobId) ?? blobId;
```

Comment:
Unrelated to Si's changes: @wes-carlson, I'm a bit confused by code in ContainerRuntime.resolveHandle() around blobs.
Shouldn't there be an await here, where we return value?
```ts
                if (handle) {
                    return {
                        status: 200,
                        mimeType: "fluid/object",
                        value: handle.get(),
                    };
                } else {
```

Reply:
I don't think it functionally makes a difference since RemoteFluidObjectHandle.get() is async, so awaiting it will await this promise too. Is there a reason we should await here?

Reply:
It works the way code is structured today. if get() were to interrogate FluidObject, then it would likely fail, as it works with promise:
```ts
                    if (response.mimeType === "fluid/object") {
                        const fluidObject: FluidObject = response.value;
                        return fluidObject;
                    }
```
It's not a big deal, but it feels wrong - requests should return results, not promises for results (as request pattern itself is async). It's equivalent to returning Promise<Promise<T>> - valid, but not useful construct.